documentation/enforce/relationships/README.md

```diff
@@ -1,9 +1,9 @@
 # Enforce conditions relationships
 When running validations against your data, there are different relationships between your rules that you need to consider. The enforce function exposes three different functions that represent possible relationships.
 
-* `allOf`  | `AND` | makes sure all rules specified are true.
-* `anyOf`  | `OR`  | makes sure at least one rule is true.
-* `noneOf` | `NOT` | makes sure no rule is true.
+* `allOf` (AND) - makes sure all rules specified are true.
+* `anyOf` (OR) - makes sure at least one rule is true.
+* `noneOf` (NOT) - makes sure no rule is true.
```

Comment:
I'd leave this change out. This area changed a lot after the recent `enforce` update.